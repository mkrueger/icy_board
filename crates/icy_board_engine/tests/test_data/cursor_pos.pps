AnsiPos 2,2
PrintLn GetX(), "x", GetY()