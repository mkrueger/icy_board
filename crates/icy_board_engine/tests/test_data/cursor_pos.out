[2;2H2x2
